frontend/src/components/IconLibrary/index.js

```diff
@@ -1,7 +1,14 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 
-import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons';
+import { faMapMarkerAlt, faCalendarAlt, faBook, faUserFriends, faHiking,
```

Comment:
Is this the only way to use the icons we need, by importing it in this component and adding it to a library?

Reply:
No, but I have a comment in PR #106 about why this method is recommended. [These docs](https://github.com/FortAwesome/react-fontawesome#build-a-library-to-reference-icons-throughout-your-app-more-conveniently) give a good explanation as to why.

Reply:
Sounds good, the added comment to the docs is appreciated.